Marlin/src/config/examples/wt150/Configuration.h

```diff
@@ -1559,16 +1559,24 @@
  * Adds the M150 command to set the LED (or LED strip) color.
  * If pins are PWM capable (e.g., 4, 5, 6, 11) then a range of
  * luminance values can be set from 0 to 255.
+ * For Neopixel LED overall brightness parameters is also available
  *
  * *** CAUTION ***
  *  LED Strips require a MOFSET Chip between PWM lines and LEDs,
  *  as the Arduino cannot handle the current the LEDs will require.
  *  Failure to follow this precaution can destroy your Arduino!
+ *  Neopixel LED is 5V powered, but linear 5V regulator on Arduino
+ *  cannot handle the current, separate 5V power supply must be used
  * *** CAUTION ***
  *
+ * LED type. This options are mutualy exclusive. Uncomment only one.
+ *
  */
+
 //#define RGB_LED
 //#define RGBW_LED
+//#define NEOPIXEL_LED
```

Comment:
Since this is essentially an option rename, there should be an entry in `SanityCheck.h` to catch `defined(NEOPIXEL_RGBW_LED)` and throw an error that the name has changed.

Reply:
Is that really needed, being that `NEOPIXEL_RGBW_LED` would be eliminated from the entire program?

Reply:
@Tannoo look into Adafruit_NeoPixel.h . The RGBW is only one from 30 options. So IMHO there is no reason to keep this one option instead of much more universal solution.

Reply:
Nevermind... I realize it's needed for old config file handling.